Octokit.Tests.Integration/Clients/RepositoryHooksClientTests.cs

```diff
@@ -276,7 +276,7 @@ public TheDeleteMethod(RepositoriesHooksFixture fixture)
                 await github.Repository.Hooks.Delete(_fixture.RepositoryOwner, _fixture.RepositoryName, _fixture.ExpectedHook.Id);
                 var hooks = await github.Repository.Hooks.GetAll(_fixture.RepositoryOwner, _fixture.RepositoryName);
 
-                Assert.Empty(hooks);
+                Assert.Equal(4, hooks.Count);
```

Comment:
Each test creates the same five hooks, so we need to verify the count of the list to be `N-1`